src/python/deepgnn/graph_engine/snark/convert.py

```diff
@@ -61,6 +63,8 @@ def __init__(
             skip_edge_sampler(bool): skip generation of edge alias tables.
             file_iterator(TextFileIterator): Iterator to yield lines of the input text file.
             debug(bool, False): Enable debug mode to disable multiprocessing and see error messages, forces worker_count=1, paritition_count=1.
+            is_temporal(bool): add timestamps to graph elements.
+            watermark(int): latest known timestamp in the graph.
```

Comment:
Can we rename watermark to temporal_watermark or something more clear / memorable.

Reply:
Watermark is a standard term https://en.wikipedia.org/wiki/Watermark_(data_synchronization)